guides/source/active_record_encryption.md

```diff
@@ -298,6 +298,42 @@ And you can disable this behavior and preserve the encoding in all cases with:
 config.active_record.encryption.forced_encoding_for_deterministic_encryption = nil
 ```
 
+### Compression
+
+The library compresses encrypted payloads by default. This can save up to 30% of the storage space for larger payloads. You can disable compression by settings `compress: false` to encrypted attributes:
+
+```ruby
+class Article < ApplicationRecord
+  encrypts :content, encryptor: ActiveRecord::Encryption::Encryptor.new(compress: false)
```

Comment:
In both cases of passing a compressor and whether to compress app owners shouldn't have to instantiate a new Encryptor. I think AR should be able to do that. Is there a reason Active Record can't do this? The API feels really clunky having to instantiate a new encryptor. It should be like this:

```ruby
encrypts :content, compress: false
```

or

```ruby
encrypts :content, compressor: ZstdCompressor
```

Reply:
@eileencodes I made `encrypts` can accept `compressor` or `compress` as arguement. https://github.com/rails/rails/pull/51735/files#diff-fc6d4f7a47751450e22af8350c689dd040d9048ee7d346b17e72b5c2157f2d2bR49-R50